acceptance/README.md

```diff
@@ -47,11 +47,12 @@ Beaker will install the following:
 
 This corresponds to installing facter and hiera stable from our internal mirror, while installing puppet SHA abcd from a git daemon on my local machine percival.  See below for details on setting up a local git daemon.
 
-Running Tests on the vcloud
+Running Tests on the vcloud (now vmpooler)
 ---------------------------
 
-In order to use the Puppet Labs vcloud, you'll need to be a Puppet Labs employee.
-Community members should see the [guide to running the tests on vagrant boxen](#running-tests-on-vagrant-boxen).
+Only applies to Pulletlabs employees. Community members should see the [guide to running the tests on vagrant boxen](#running-tests-on-vagrant-boxen).
```

Comment:
Typo "Pulletlabs"

Reply:
I see that came in from the 5.5.x branch....

Reply:
I'm adding a maint commit on top to fix the typo.

Reply:
Actually, it can be its own PR to get merged up.